stripe/src/main/java/com/stripe/android/model/PaymentIntent.kt

```diff
@@ -150,6 +150,7 @@ data class PaymentIntent internal constructor(
     /**
      * The URL you must redirect your customer to in order to authenticate the payment.
      */
+    @Deprecated("use {@link #nextActionData}")
```

Comment:
Should we point directly to the subclass that represents redirect? Also is there an opportunity to use `ReplaceWith`?

Reply:
added a ReplaceWith to point to the subclass